### ISSUE TITLE
Window gets moving sideways when reopening during the same session

### ISSUE THREAD

**Issue by skunkyevil** [highfestiva/finplot#421]
First of all thank you for the great tool that saves a huge amount of time for not reinventing a wheel 👍 

I use Finplot in Jupyter notebook mostly.
With the first fplt.show() window position is fine.
With next fplt.show() window appears slightly shifted to rightwards and a little bit to the downwards.
And so on and so forth, so at some point window appears completely out of the screen, so I have to restart kernel, load all my data and so on.

To confirm this behavior without Jupyter, I just modified `pandas-df-plot.py` example:

```
#!/usr/bin/env python3

import finplot as fplt
import pandas as pd

df = pd.DataFrame({'A':[1,2,3,5], 'B':[2,4,5,3], 'C':[3,6,11,8], 'D':[1,1,2,2], 'labels':list('ABCD')}, index=[1606086000, 1606086001, 1606086002, 1606086003])

df.plot('A')
df.plot('A', 'labels', kind='labels', color='#660')
df.plot('C', kind='scatter')
(1-df.B).plot(kind='scatter')
df.plot(kind='candle').setZValue(-100)
(df.B-4).plot.bar()

fplt.show()


df.plot('A')
df.plot('A', 'labels', kind='labels', color='#660')
df.plot('C', kind='scatter')
(1-df.B).plot(kind='scatter')
df.plot(kind='candle').setZValue(-100)
(df.B-4).plot.bar()

fplt.show()

```

**Comment by highfestiva** [comment]
Ah, hadn't thought of that. Fix in 15059aafb.

**Comment by highfestiva** [comment]
Correct, it is an old-fashioned way of making sure the user sees there are more windows opened.
